src/benchmarks/real-world/ILLink/ILLinkBenchmarks.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFrameworks Condition="'$(TargetFrameworks)' == ''">net6.0;net7.0</TargetFrameworks>
```

Comment:
You need to add this line above the current `<TargetFrameworks>` tag

`<TargetFrameworks>$(PERFLAB_TARGET_FRAMEWORKS)</TargetFrameworks>`

This will ensure in the lab that you are only building against the one TFM that you care about at that time. It will prevent the kind of errors that we are seeing now.